openquakeplatform/openquakeplatform/static/js/irv_viewer_d3_tree.js

```diff
@@ -89,60 +89,137 @@
             });
         }
 
-        var nodeEnter;
-        function updateButton(){
-            $('#projectDefWeightDialog').append('<br/><br/><button type="button" id="update-spinner-value">Update</button>');
-            $('#update-spinner-value').click(function() {
-                pdTempWeights = [];
-                pdTempWeightsComputed = [];
-                pdTempInverters = [];
-
-                // Get the values of the spinners and inverters
-                for (var i = 0; i < pdTempSpinnerIds.length; i++) {
-                    var isInverted = $('#inverter-' + pdTempSpinnerIds[i]).is(':checked');
-                    var spinnerValue = $('#'+pdTempSpinnerIds[i]).val();
-
-                    pdTempInverters.push(isInverted);
-                    pdTempWeights.push(spinnerValue);
-                }
+        $('#submitPD').attr('disabled',true);
+
+        function saveProjData() {
+            $('#saveBtn').click(function() {
+                $('#saveState-spinner').hide();
+                var pdLicense = sessionProjectDef.license;
+                var pdLicenseName = sessionProjectDef.license.substring(0, sessionProjectDef.license.indexOf('('));
+                var pdLicenseURL = sessionProjectDef.license.split('(')[1];
+                pdLicenseURL = pdLicenseURL.replace(')', '');
+
+                $('#saveState-spinner').hide();
+                $('#saveStateDialog').dialog('open');
+                $('#licenseName').empty();
+                $('#licenseURL').empty();
+                $('#inputName').empty();
+                $('#inputName').append('<p>The current title is: '+ projectDef.title +'</p><p> <input id="giveNamePD" type="text" name="pd-name"></p><br><br>');
+                $('#licenseName').append('<p>This project has been created using the '+ pdLicenseName +' license</p>');
+                $('#licenseURL').append('<a class="btn btn-blue btn-xs" target="_blank" href="'+ pdLicenseURL +'">Info</a></br>');
+
```

Comment:
Please find all occurrences of `</br>` and replace them with `<br>` or `<br />` (choose the one you used in the other cases pointed out by Daniele)